doc/indexing.rst

```diff
@@ -395,6 +395,22 @@ These methods may also be applied to ``Dataset`` objects
     ds = da.to_dataset(name="bar")
     ds.isel(x=xr.DataArray([0, 1, 2], dims=["points"]))
 
+Vectorized indexing may be used to extract information from the nearest
+grid cells of interest, for example, the nearest climate model grid
+cells to a collection specified weather station latitudes and longitudes.
+
+.. ipython:: python
+
+    ds = xr.tutorial.open_dataset("air_temperature")
+
+    # Define target latitude and longitude (where weather stations might be)
+    tgt_lat = xr.DataArray(np.linspace(40, 45, num=6), dims="points")
```

Comment:
I think it's preferable to use the existing dataset in this file.

This
```
 da.isel(
    x=xr.DataArray([0, 1, 6], dims="z", coords={"z": ["a", "b", "c"]}),
    y=xr.DataArray([0, 1, 0], dims="z"),
)
```
could be copied and modified to illustrate `method="nearest"`